### ISSUE TITLE
BAWDEN; UPTIME DATA should get updated to remove non-existent hosts

### ISSUE THREAD

**Issue by eswenson1** [PDP-10/its#1469]
Currently, BAWDEN; UPTIME DATA, used by the UPTIME daemon, has entries for the following hosts:  ML, MC, MX, AI, and MD.  None of these hosts exists any more, and UPTIME will needlessly hang waiting on TCP or CHAOS connections to the MLSLV devices on those systems.  Furthermore, it will never update this data file because none of these hosts will ever respond.  

It would be far more useful to populate this data file with hosts that do exist (NO, ES, UP, LX, DB, etc.)

It turns out that UPTIME will result in dead MLDEV jobs if any of the hosts specified in BAWDEN; UPTIME DATA are not listed in the running system's host table.  On ES, I did not have an entry for MD, and this caused an accumulation of dead MLDEV jobs.  While I could add MD to the host table, this seemed the wrong thing to do.  MD will never respond again, right?

**Comment by larsbrinkhoff** [comment]
I think the original file is interesting, because it could reflect the true dates when the machines were shut down.  But it could be renamed e.g. UPTIME ODATA.

**Comment by eswenson1** [comment]
I agree. That’s what I did on ES — renamed original. We should do this on DB too.
